packages/IMAPClient-Core.package/ICEmail.class/instance/modifyFlagFlagged.st

```diff
@@ -0,0 +1,12 @@
+fetch body
```

Comment:
Couldnt we join both methods `modifyFlagFlagged` and `modifyFlagSeen` into a `modifyFlag: aFlag`. Both share 90% the same code :)

Reply:
Very much agree.